Runtime/TimeManager.cs

```diff
@@ -79,7 +84,7 @@ private void Awake()
 
         private void Update()
         {
-            if(Input.GetKeyDown(togglePauseKey) == true)
+            if(isPauseKeyPressed())

```

Comment:
Based on the example mentioned in the contribution doc, to reduce ambiguity, this can be rewritten as `if(isPauseKeyPressed()  == true)`